### ISSUE TITLE
graph line error at the coolprop plot

### ISSUE THREAD

**Issue by myjesus123** [CoolProp/CoolProp#1650]
```
--------------------------------------------------------
  File "C:\Users\lge\AppData\Local\Programs\Python\Python35\lib\site-packages\CoolProp\Plots\Plots.py", line 217
    warnings.warn("Detected an incomplete phase envelope, fixing it numerically.")
UserWarning: Detected an incomplete phase envelope, fixing it numerically.
---------------------------------------------------------
```
After do coolprop plotting, I got above error message and distorted graph line.
is there anyone who could solve the above issued program?
please help me.

**Comment by ibell** [comment]
I'm afraid that the plotting code in CoolProp is quite broken, and we don't have anyone to fix it right now :(.  Sorry.

**Comment by myjesus123** [comment]
It is so sad to see your answer. Is there any chance to solve this problem?

**Comment by ibell** [comment]
See the other issue.  Basically, the answer for now is no...  I'm really
sorry.

**Comment by myjesus123** [comment]
Yeah, but suddenly I made it by iterating install and uninstall job. 
i think it is awesome to make hard training for me, super-beginner.

**Comment by d-e-t** [comment]
I have this problem with Python 3, but not with Python 2.

I would love to see the plotting fixed for Python3 and extended -- it is so useful to have...

**Comment by ibell** [comment]
That's certainly true, but we have currently nobody to work on it, so
there's your problem.  See also
https://groups.google.com/d/msg/coolprop-users/uyhIMHmaoko/rrJCInMzCgAJ

**Comment by myjesus123** [comment]
I concluded that this problem is mainly related with PIP install figure.
As mention with above program, troublesome program is not coolprop but surely matplotlib.
if you upgrade matplotlib version or re-install, you could solve it perhaps, i think that.
um..

**Comment by d-e-t** [comment]
@ibell - I had not heard of your move to NIST and leaving CoolProp -- congratulations!  Bummer that you're moving away from CoolProp, but you've certainly left it in a very good useful state, so thank you!

**Comment by Immortalyzy** [comment]
The problem might be fixed by installing SciPy.

**Comment by ibell** [comment]
Closing as part of an issue-tracker cleanup — this issue has had no activity in over two years.

*To be transparent: this comment and the close action are being performed by Claude (Anthropic's AI coding assistant), running on behalf of @ibell. Ian reviewed and approved the bulk-close criteria; I am the one executing the action.*

If you would like this issue reconsidered, please feel free to **re-open it** — a fresh comment with current behaviour against the latest CoolProp release is the most useful way to revive the discussion.

— Claude (acting for @ibell)
